### PR TITLE
topology dump cli command

### PR DESCRIPTION
**Pull request by raghavendra-talur**
Closes: #753

### CODE REVIEW

File: pkg/glusterfs/api/types.go [hunk 1]
```diff
@@ -102,21 +102,31 @@ type NodeInfo struct {
...
+type Node struct {
+	NodeName NodeName `json:"node"`
+	Devices  []string `json:"devices"`
+}
+
 type NodeInfoResponse struct {
 	NodeInfo
 	State       EntryState           `json:"state"`
-	DevicesInfo []DeviceInfoResponse `json:"devices"`
+	DevicesInfo []DeviceInfoResponse `json:"deviceInfo"`
```
Comment (lpabon) [inline pkg/glusterfs/api/types.go]: Api changes can only happen when absolutely necessary. Here the patch is changing the API.

**Comment by lpabon** [comment]
@raghavendra-talur Try doing this using the existing APIs instead.

File: pkg/glusterfs/api/types.go (hunk 1 above)
Comment (raghavendra-talur) [inline pkg/glusterfs/api/types.go]: Makes sense. Will update the patch.

**Comment by obnoxxx** [comment]
@raghavendra-talur I think we can close this, right? It's been superseded by the export/import command (PR #959).

**Comment by raghavendra-talur** [comment]
@obnoxxx not really, this is a dump in the format of topology file. Which a user might edit and load again to extend the cluster. Working on this, will update the patch.

**Comment by obnoxxx** [comment]
@raghavendra-talur any intentions to update this?

**Comment by phlogistonjohn** [comment]
Closing PRs older than 2018. No expectation this will be continued. Feel free to reopen if things change.

**Pull request closed without merging**